include/arch/x86/linker.ld

```diff
@@ -136,6 +136,15 @@ SECTIONS
 
 	__data_ram_end = .;
 
+#ifdef CONFIG_FIRMWARE_VERSION_STRING
+	_version_rom_start = FIRMWARE_FLASH_END - VERSION_SECTION_SIZE;
+
+	_VERSION_SECTION_NAME _version_rom_start :
```

Comment:
Looks good. Let's use SECTION_PROLOGUE to match the rest of the file though

    SECTION_PROLOGUE(_VERSION_SECTION_NAME, _version_rom_start (OPTIONAL),)